coalib/settings/ConfigurationGathering.py

```diff
@@ -385,6 +385,20 @@ def gather_configuration(acquire_settings,
     local_bears, global_bears = fill_settings(sections,
                                               acquire_settings,
                                               log_printer)
+
+    exclude = False
+    for x in arg_list:
+        if x == '-ex' or x == '--exclude-sections':
+            exclude = True
+        elif exclude and '-' not in x:
```

Comment:
What if the section name contain `-`? This could be avoided if we use `arg_parser`.